workspaces/marketplace/packages/cli/src/commands/generate/index.ts

```diff
@@ -0,0 +1,293 @@
+        path.dirname(defaultDynamicPluginsConfig),
+        wrapperDir,
+      );
+      const packageJSONPath = path.join(wrapperPath, 'package.json');
+      const packageJSON = fs.readJsonSync(packageJSONPath) as PackageJson;
+
+      // parse packgeName
+      const packageName =
+        guessPackageFromDependencies(
+          packageJSON.name,
+          packageJSON.dependencies,
+        ) || packageJSON.name;
+
+      // parse links
+      const links: EntityLink[] = [];
+      if (packageJSON.homepage) {
+        links.push({ url: packageJSON.homepage, title: 'Homepage' });
+      }
+      if (packageJSON.bugs) {
+        const bugs = packageJSON.bugs as PackageJsonBugs;
+        if (bugs.url) {
+          links.push({ url: bugs.url, title: 'Bugs' });
+        } else {
+          links.push({ url: packageJSON.bugs as string, title: 'Bugs' });
+        }
+      }
+      let sourceLocation: string = '';
+      if (packageJSON.repository) {
+        const repo = packageJSON.repository as PackageJsonRepository;
+        const url: EntityLink = { title: 'Source Code', url: '' };
+        if (repo.url) {
+          // try to create a valid github link
+          if (repo.url.startsWith('https://github.com')) {
+            url.url = `${repo.url}/tree/main/${repo.directory}`;
+          } else {
+            url.url = `${repo.url}/${repo.directory}`;
+          }
+        } else {
+          url.url = packageJSON.repository as string;
+        }
+        links.push(url);
+        sourceLocation = `url ${url.url}`;
+      }
+
+      // parse author
+      let author: string = '';
+      if (packageJSON.author) {
+        const a = packageJSON.author as PackageJsonPerson;
+        if (a.name) {
+          author = a.name;
+          author += a.email ? ` <${a.email}>` : '';
+          author += a.url ? ` (${a.url})` : '';
+        } else {
+          author = packageJSON.author as string;
+        }
+      }
+
+      // parse partOf
+      const partOf: string[] = [];
+      if (packageJSON.backstage?.pluginId) {
+        if (packageJSON.backstage?.role === 'backend-plugin-module') {
+          const scope = getScope(packageName);
+          const module = getModuleName(packageName);
+          let pof = '';
+          if (scope) {
+            pof = `${scope}-`;
+          }
+          if (module) {
+            pof += `${module}-`;
+          }
+          pof += `${packageJSON.backstage.pluginId}-module`;
+          partOf.push(pof);
+        } else {
+          partOf.push(packageJSON.backstage.pluginId);
+        }
+      }
+
+      // parse config examples
+      const appConfigExamples: AppConfigExample[] = [];
+      if (plugin.pluginConfig) {
+        appConfigExamples.push({
+          title: 'Default configuration',
+          content: plugin.pluginConfig,
+        });
+      }
+
+      const entity: MarketplacePackage = {
+        kind: MarketplaceKinds.package,
+        apiVersion: MARKETPLACE_API_VERSION,
```

Comment:
Personal opinion. You must not change it, only if you agree on that ;)

```suggestion
        apiVersion: MARKETPLACE_API_VERSION,
        kind: MarketplaceKinds.package,
```